### PR TITLE
docs: fix cleanup docs

### PR DESCRIPTION
**Pull request by stavros-liaskos**
### What is accomplished by your PR?
If the user follows the docs [here](https://vue-starter.herokuapp.com/docs/guide/clean-up.html#code), he will delete code in `./src/app/state.ts` that will break the application (note that particles are not floating anymore and burger menu is not clickable). More specifically, all the following lines should *NOT* be deleted but rather only the middle one:
```vue
counter: {
    ...CounterDefaultState(), // delete only this line
  },
```

### CODE REVIEW

File: docs/guide/clean-up.md [hunk 1]
```diff
@@ -45,9 +45,7 @@ import { CounterDefaultState, ICounterState } from './counter/state';
 
 ...
 
-  counter: {
     ...CounterDefaultState(),
-  },
```
Comment (devCrossNet) [inline docs/guide/clean-up.md]: hey, if you only remove `...CounterDefaultState(),` you would still have `counter:{}` in the state. This should also be removed. Are you sure the app wont work by removing the whole `counter` state entry?

**Comment by devCrossNet** [comment]
I just tried it out by myself on the latest copy of the vue-starter master and it seems to work. Did you get an error?

**Comment by stavros-liaskos** [comment]
I've tested it in different machines. It doesn't work. In order to view the bug, you have to restart the server make sure that you are not viewing a cached version. The result will be: the page will load but burger menu and particles are not working anymore

**Comment by devCrossNet** [comment]
@stavros-liaskos I will try it again. Yesterday evening I deleted all the stuff as described in the docs and then I started the dev server and everything was ok

**Comment by devCrossNet** [comment]
I was able to reproduce it. the solution is, we have to delete 2 additional lines. would be nice if you can add this to your PR:

./src/app/store.ts

```js
...
  delete localState.counter.incrementPending;
  delete localState.counter.decrementPending;
...
```

**Pull request merged**